templates/kickstart.j2

```diff
@@ -1,6 +1,11 @@
 install
 text
+reboot
+skipx
+
+{% if hostvars[groups[item][0]]['dhcp_kickstart_disable_ipv6'] is True %}
 network --noipv6
```

Comment:
If this is made conditional, shouldn't there be two cases

`network`

and

`network --noipv6`

?

Or is it default to run `network` if it's missing?